pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -212,7 +212,7 @@ def snipe(self, pokemon):
         last_position = self.bot.position[0:2]
         self.bot.heartbeat()
         self._teleport_to(pokemon)
-        catch_worker = PokemonCatchWorker(pokemon, self.bot)
+        catch_worker = PokemonCatchWorker(pokemon, self.bot, self.config)
```

Comment:
This likely doesn't do what you're expecting. You're passing the MoveToMapPokemon's config to the PokemonCatchWorker. None of the keys that the catch worker is expecting will be there. You can send an empty dict to use the defaults if that's what is desired.

If you want to specify different values for the catching config, you need to add the keys to the MoveToMapPokemon config section.